app/views/finders/_option_select_facet.html.erb

```diff
@@ -1,12 +1,11 @@
 <% cache_if option_select_facet.cacheable?, option_select_facet.cache_key do %>
-  <% close_facet = option_select_facet.unselected? && ((option_select_facet_counter > 0) || option_select_facet.close_facet?) %>
   <%= render partial: 'components/option-select', locals: {
     :key => option_select_facet.key,
     :title => option_select_facet.name,
     :aria_controls_id => "js-search-results-info",
     :options_container_id => option_select_facet.key,
     :options => option_select_facet.options("js-search-results-info", option_select_facet.key),
-    :closed_on_load => close_facet,
+    :closed_on_load => !option_select_facet.open_on_load?,
```

Comment:
Would it be possible to add a method that returns the value of `!option_select_facet.open_on_load?`?  To someone looking at the code for the first time, it might not be obviously clear what `!option_select_facet.open_on_load?` is doing, so having a named method would make that clearer.

Reply:
I'm not sure what you mean - are you suggesting adding a `closed_on_load?` method to `option_select_facet.rb` like so:

```ruby
def closed_on_load?
  !open_on_load?
end
```

...and then updating this line to `:closed_on_load => option_select_facet.closed_on_load?` ...?

Reply:
Personally, I'd create a `closed_on_load?` method.  But I guess it's down to personal preference.  I'll leave it up to you to decide which you prefer.

Reply:
Hmm, Andrew said the same thing here: https://github.com/alphagov/finder-frontend/pull/1063#pullrequestreview-230542295

And I agree it would make this one line clearer. But it will also make other lines harder to read, e.g.

```ruby
def at_least_one_facet_selected?(facets_to_ensure)
  facets_to_ensure.any?(&:open_on_load?)
end
```

...becomes:

```ruby
def at_least_one_facet_selected?(facets_to_ensure)
  !facets_to_ensure.all?(&:closed_on_load?)
end
```

Unless we _add_ a `closed_on_load?` method in addition to `open_on_load?`...? Then we could pick and choose whatever reads best in the context. Or are you suggesting we pick `closed_on_load?` and use that exclusively?

(I don't feel strongly either way. I mainly renamed it to `open_on_load?` to make that method easier to mentally parse, as it was much easier when having to deal with the `>10 options` requirement too. But now that's gone there's not much in it!)

Reply:
Have chatted on Slack, agreed we should add additional methods.